src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -22,33 +22,40 @@ public class LogsCenter {
     private static final int MAX_FILE_SIZE_IN_BYTES = (int) (Math.pow(2, 20) * 5); // 5MB
     private static final String LOG_FILE = "addressbook.log";
     private static Level currentLogLevel = Level.INFO;
-    private static final Logger logger = LogsCenter.getLogger(LogsCenter.class);
-    private static FileHandler fileHandler;
-    private static ConsoleHandler consoleHandler;
+    // The logger that will be used as the parent of all other loggers created by this class.
+    private static final Logger logger;
+    private static Logger baseLogger;
+
+    static {
+        setBaseLogger();
+        logger = LogsCenter.getLogger(LogsCenter.class);
+    }
 
     /**
-     * Initializes with a custom log level (specified in the {@code config} object)
-     * Loggers obtained *AFTER* this initialization will have their logging level changed<br>
-     * Logging levels for existing loggers will only be updated if the logger with the same name
-     * is requested again from the LogsCenter.
+     * Initializes loggers with the log level specified in the {@code config} object. Applies to all loggers created
+     * using {@link #getLogger(String)} and {@link #getLogger(Class)} methods except for those that are manually set.
      */
     public static void init(Config config) {
         currentLogLevel = config.getLogLevel();
-        logger.info("currentLogLevel: " + currentLogLevel);
+        logger.info("log level set as: " + currentLogLevel);
```

Comment:
```suggestion
        logger.info("log level will be set as: " + currentLogLevel);
```
Because it is not set yet. Alternatively, move to the bottom of the method (i.e., _after_ setting the log level).